src/uwtools/drivers/cdeps.py

```diff
@@ -54,7 +53,7 @@ def atm_stream(self):
         yield self._taskname(f"stream file {fn}")
         path = self._rundir / fn
         yield asset(path, path.is_file)
-        template_file = self._driver_config["atm_streams"]["template_file"]
+        template_file = self.config["atm_streams"]["template_file"]
```

Comment:
A LOT of lines like this follow, so hopefully you can visually pattern-match on and ignore them after this. Old references any of `._driver_config`, `._confg`, `._config_full`, `.config`, or `.config_full` are updated to use the new versions (except `._driver_config`, which is removed) as described in the PR Description.

Reply:
`.config` and `.config_full` were introduced on `main` 3 weeks ago and are not in a release yet, so no one should be relying on them. Are you aware of any use cases?